spec/fabricators/following_fabricator.rb

```diff
@@ -0,0 +1,3 @@
+Fabricator(:following) do
```

Comment:
You can scan the docs for the gem in about 5 minutes, if you are so inclined. A lot of good information here: http://www.fabricationgem.org/

The "Defining Fabricators" tab applies to this specifically.